### PR TITLE
Add doc for english LM.

### LINKED ISSUE #294: Add doc for english LM training

### PR DESCRIPTION
**Pull request by pkuyym**
fix #294

### CODE REVIEW

File: deep_speech_2/README.md [hunk 1]
```diff
@@ -219,6 +219,18 @@ sh download_lm_ch.sh
 ```
 If you wish to train your own better language model, please refer to [KenLM](https://github.com/kpu/kenlm) for tutorials.
 
+Here we provide some tips to show how we prepearing our english and mandarin language models.
+
+#### English LM
+
+The english corpus is from the [Common Crawl Repository](http://commoncrawl.org) and you can download it from [statmt](http://data.statmt.org/ngrams/deduped_en). We use part en.00 to train our english languge model. There are some preprocessing steps before training:
+
+  * Characters which not in [A-Za-z0-9\s'] are removed and arabic numbers are converted to english numbers like 1000 to one thousand.
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: remove `which`
why `\s` ? whitespace?

File: deep_speech_2/README.md [hunk 2]
```diff
@@ -219,6 +219,18 @@ sh download_lm_ch.sh
...
+  * Repeated whitespace are squeezed to one and the beginning whitespace are removed. Notice that all transcriptions are lowercase, so all characters are converted to lowercases.
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: --> beginning and trailing ?
whitespace 的复数形式是？
两处lowercase的单复数形式？

File: deep_speech_2/README.md [hunk 3]
```diff
@@ -219,6 +219,18 @@ sh download_lm_ch.sh
...
+  * Top 400000 words by frequency are selected to build the vocabulary and all words not in the vocabulary are replaced with 'UNKNOWNWORD'.
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: ---> Top 400,000 most frequent words are ....and the rest are replaced with ...

File: deep_speech_2/README.md [hunk 4]
```diff
@@ -219,6 +219,18 @@ sh download_lm_ch.sh
...
+
+Now the preprocessing is done and we get a clean corpus to train the language model. Our released language model are pruned by '0 1 1 1 1'. To save disk storage we convert the arpa file to 'trie' binary file with parameters '-a 22 -q 8 -b 8'.
```
Comment (pkuyym) [inline deep_speech_2/README.md]: Our released language model are pruned by '0 1 1 1 1' and the max order of n-gram is 5.

Comment (xinghai-sun) [inline deep_speech_2/README.md]: ”0 1 1 1 1“ 不熟悉kenlm的人看比较费解，能给出是哪个参数并稍微解释下吗。"-a -q -b"同上。

File: deep_speech_2/README.md [hunk 5]
```diff
@@ -219,6 +219,18 @@ sh download_lm_ch.sh
 ```
 If you wish to train your own better language model, please refer to [KenLM](https://github.com/kpu/kenlm) for tutorials.
 
+Here we provide some tips to show how we prepearing our english and mandarin language models.
+
+#### English LM
```
Comment (xinghai-sun) [inline deep_speech_2/README.md]: Prepare Your Own English LM ? 以和前面区分开来？
并且在前面部分的最开始加上 #### Download LMs

**Review by xinghai-sun — CHANGES_REQUESTED** [review]

File: deep_speech_2/README.md (hunk 2 above)
Comment (pkuyym) [reply deep_speech_2/README.md]: trailing的whitespace影响不大，比如句尾的换行符，没有必要去掉

**Review by xinghai-sun — APPROVED** [review]

**Pull request merged**